pkg/ansible/jobgenerator.go

```diff
@@ -208,6 +205,31 @@ func (r *jobGenerator) GeneratePlaybookJob(name string, hardware *clusteroperato
 		}
 	}
 
+	containers := make([]kapi.Container, len(playbooks))
+	for i, playbook := range playbooks {
+		playbookPath := path.Join(openshiftAnsibleContainerDir, playbook)
+		envForPlaybook := make([]kapi.EnvVar, len(env)+1)
+		copy(envForPlaybook, env)
+		envForPlaybook[len(env)] = kapi.EnvVar{
+			Name:  "PLAYBOOK_FILE",
+			Value: playbookPath,
+		}
+		containers[i] = kapi.Container{
+			Name:            containerNameForPlaybook(playbook),
+			Image:           r.image,
+			ImagePullPolicy: r.imagePullPolicy,
+			Env:             envForPlaybook,
+			VolumeMounts:    volumeMounts,
+		}
+	}
+
+	podSpec := kapi.PodSpec{
+		DNSPolicy:     kapi.DNSClusterFirst,
+		RestartPolicy: kapi.RestartPolicyNever,
+		Containers:    containers,
```

Comment:
This is clever, I don't think I'd have thought to run all the containers in one pod. 

This tested fine for me, but what would happen if one of the jobs failed? Is it going to re-try every container in the pod? Or just the one that failed.

Reply:
If the pod is reused, it will retry just the containers that failed. In the current state of jobs, however, the pods are re-created rather than reused. If we take #114, then only the playbooks that failed will be retried.

Reply:
Forgot about that PR, I'll go get it in.